test/kpod_mount.bats

```diff
@@ -13,41 +13,32 @@ function setup() {
 }
 
 @test "mount" {
-    skip "Test needs to be converted to kpod run"
-    start_crio
-    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
-    echo "$output"
-    [ "$status" -eq 0 ]
-    pod_id="$output"
-    run crioctl image pull "$IMAGE"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} create $BB ls
     echo "$output"
     [ "$status" -eq 0 ]
     ctr_id="$output"
-    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} mount $ctr_id
```

Comment:
These bash -c might come back to haunt you.  Centos tests are blowing up without them right now.

Reply:
I have no idea why, but the bash -c prevents these on Centos.

# Incorrect Usage. flag provided but not defined: -root /test.dir/tmp.uI7BV50XI4/crio  --runroot /test.dir/tmp.uI7BV50XI4/crio-run --runtime /usr/bin/runc --conmon /usr/libexec/crio/conmon

Reply:
funny I get that locally with bash -c
```
bash -c /home/umohnani/go/src/github.com/projectatomic/libpod/bin/kpod --root /test.dir/tmp.3Dm8aTs496/crio --runroot /test.dir/tmp.3Dm8aTs496/crio-run --runtime /bin/runc --conmon /home/umohnani/go/src/github.com/projectatomic/libpod/bin/conmon mount a07f2c6b8a942a30b6b272f07016fe75743bfe17347d1ef6c5919778e0814071
```

Reply:
so that happens when create is done without bash -c and the other commands are with bash -c. If I do create with bash -c it just hangs. I will push it, maybe it won't hang on the other tests.